compute/api/Compute.Samples/SetUsageExportBucketAsync.cs

```diff
@@ -0,0 +1,54 @@
+﻿/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// [START compute_usage_report_set]
+// [START compute_instances_verify_default_value]
```

Comment:
Is it OK to have the same region tag in two different files? We want to include SetUsageExportBucket and GetUsageExportBucket in one sample (compute_instances_verify_default_value), but I am not sure if this will work correctly. Can you confirm?

Reply:
Hmm now that you mention it I'm not certain if the code can be picked up from two different files. Would it be terrible to have two distinct region tags, one for checking input and another for checking output? Our general policy is to not have code for different samples on the same file.

Reply:
It seems we can include region_tags from different files, but it will look a bit different. We are now discussing if we actually need the tag 'compute_instances_verify_default_value' as we may decide to use only 'compute_usage_report_set' and 'compute_usage_report_get' - this way there will be no such problem. I should have some decisions tomorrow and will let you know.

Reply:
@amanda-tarafa We decided to refactor the snippets and get rid of the ‘compute_instances_verify_default_value’ region tag. Instead we will use ‘compute_usage_report_set’ and ‘compute_usage_report_get’ only. So the only thing for you to do is to remove the unnecessary tag and you should be OK to merge your code.

Reply:
Looks like this is still pending?